upd: intro note for desktop snippets (#706)
JetBrains/kotlin-multiplatform-dev-docs: topics/compose/compose-desktop-scrollbars.md

You can apply scrollbars to scrollable components. The scrollbar and scrollable components share a common state to
synchronize with each other.

> > Each provided snippet is a full, runnable desktop app. To try it:

Reviewer: ```suggestion
> Each provided snippet is a runnable desktop app. To try them out:
```

Author: Some pages have only one snippet

Reviewer: I just mean that "runnable" and "full" are redundant. Besides, the measure of "fullness" is unclear — without context, if it runs, it's full, no?

> > 1. Create a Compose Multiplatform project using the

Reviewer: This probably needs "with a desktop target" somewhere.

Author: The note is already too long 🥲
At least, there is a desktop target in the online wizard link

Reviewer: That's also the argument for not worrying about a couple extra words :) but I don't think it matters much.

> [Kotlin Multiplatform IDE plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform)
> or the [online wizard](https://kmp.jetbrains.com/?android=true&desktop=true&includeTests=false).

> > 2. Open the main desktop entry-point file. For example, `desktopApp/src/main/kotlin/com/example/my_desktop_app/main.kt`.

Reviewer: By definition there is only one entry point, so.
```suggestion
> 2. Open the desktop entry-point file. For example, `desktopApp/src/main/kotlin/com/example/my_desktop_app/main.kt`.
```
